Classes/Core/AppController.m

```diff
@@ -3496,9 +3371,9 @@ - (void)updateDisplay:(NSTimer *)aTimer
         
 		if(annotationPlayback && selectedAnnotation && [selectedAnnotation isDuration])
 		{
-            QTTime currentTime = [mMovie currentTime];
-            QTTime normalizedStart = QTMakeTimeScaled([selectedAnnotation startTime], currentTime.timeScale);
-			if(QTTimeCompare(currentTime,[selectedAnnotation endTime]) == NSOrderedDescending)
+            CMTime currentTime = [mMovie currentTime];
+            CMTime normalizedStart = CMTimeConvertScale([selectedAnnotation startTime], currentTime.timescale, kCMTimeRoundingMethod_Default); // TODO: Check if QTMakeTimeScaled is correctly replacesd by CMTimeConvertScale
```

Comment:
Is this replacement correct? I can't find any documentation on `QTMakeTimeScaled` to ensure that it did the same as `CMTimeConvertScale` does.

Reply:
Note that there is the same todo in `TranscriptView.m`.

Reply:
@Y0hy0h I looked into this a bit, and this is the correct replacement.